.github/workflows/ci-extends.yml

```diff
@@ -0,0 +1,81 @@
+---
+
+name: ci-extends
```

Comment:
If it runs only the saucelabs integration, maybe `ci-saucelabs` can be more accurate than `ci-extends`?